lib/bamboo/mailer.ex

```diff
@@ -1,55 +1,18 @@
 defmodule Bamboo.Mailer do
   @moduledoc """
-  Sets up mailers that make it easy to configure and swap adapters.
+  Functions for delivering emails using adapters and delivery strategies.
 
-  Adds `deliver_now/1` and `deliver_later/1` functions to the mailer module it is used by.
+  Adds `deliver_now/1` and `deliver_later/1` functions to the mailer module it
+  is used by.
 
-  ## Bamboo ships with the following adapters
+  Bamboo [ships with several adapters][available-adapters]. It is also possible
+  to create your own adapter.
 
-  * `Bamboo.MandrillAdapter`
-  * `Bamboo.LocalAdapter`
-  * `Bamboo.TestAdapter`
-  * or create your own with `Bamboo.Adapter`
+  See the ["Getting Started" section of the README][getting-started] for an
```

Comment:
I wonder if it might be good to keep the complete instructions for creating a mailer here, in addition to the README, so that the Hex docs are complete by themselves? Thoughts?

Reply:
I like that idea as well. If we're concerned about maintaining two versions of documentation, we could have the readme link out to the official docs.

Reply:
Thanks for the feedback. I'll pull the instructions back in here and consider linking from the README to this. As you all think it's valuable to have here, my initial inclination is to duplicate the instruction as I think it's helpful for someone to see it laid out in the README as that's the first place they will likely go in trying to get started on a project. I'll play around with it and see.

Reply:
Having looked at, I think some duplication is the best solution. I've re-used the code examples from the README while updating the text flow to fit better within this context.

Reply:
Sweet, works for me! Thanks for doing some investigation 🎉